apps/mull-api/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,38 @@
+require('dotenv').config();
+
 export const environment = {
   production: true,
+
+  db: {
+    host: process.env.DB_HOST,
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+  },
+  auth: {
+    google: {
+      clientId: process.env.GOOGLE_CLIENT_ID,
+      clientSecret: process.env.GOOGLE_SECRET,
+    },
+    facebook: {
+      clientId: process.env.FACEBOOK_CLIENT_ID,
+      clientSecret: process.env.FACEBOOK_SECRET,
+    },
+    twitter: {
+      consumerId: process.env.TWITTER_CONSUMER_ID,
+      consumerSecret: process.env.TWITTER_CONSUMER_SECRET,
+    },
+  },
+  session: {
+    secret: process.env.SESSION_SECRET,
+  },
+  jwt: {
+    accessSecret: process.env.ACCESS_TOKEN_SECRET,
+    refreshSecret: process.env.REFRESH_TOKEN_SECRET,
+  },
+  client: {
+    baseUrl: 'https://www.mullapp.com',
+  },
+  backend: {
+    url: 'https://www.mullapp.com',
```

Comment:
I'm curious if the baseUrl and url should be in the .env

Reply:
Since these environment variables won't be changing at all and they aren't secrets that need to be pulled in from a .env file, I think this is just fine.